x-pack/plugins/maps/common/constants.js

```diff
@@ -7,6 +7,9 @@
 export const GIS_API_PATH = 'api/maps';
 
 export const MAP_SAVED_OBJECT_TYPE = 'map';
+export function createMapPath(id) {
```

Comment:
This isn't really the kind of export I'd expect to see in a contants.js - maybe have it in a different helper file under `/common/` ?

Reply:
I really don't want to create a new file for a single function. This is part of the Maps application and we will refactor when there are more functions like this.